extra/Dockerfile.armv7-armhf

```diff
@@ -65,6 +66,6 @@ ADD extra/motioneye.conf.sample /usr/share/motioneye/extra/
 
 CMD test -e /etc/motioneye/motioneye.conf || \    
     cp /usr/share/motioneye/extra/motioneye.conf.sample /etc/motioneye/motioneye.conf ; \
-    /usr/local/bin/meyectl startserver -c /etc/motioneye/motioneye.conf
+    /usr/local/bin/meyectl startserver -c /etc/motioneye/motioneye.conf -d
```

Comment:
Do we really want `-d` here?

Reply:
It was necessary for me. Otherwise motioneye told me during network camera registration: "Protocol "rtsp" is not supported by libcurl".

Reply:
Maybe I just need more time to investigate, what happened there.